packages/webpack-extraction-plugin/__fixtures__/babel/basic/output.ts

```diff
@@ -0,0 +1,11 @@
+import { __styles, __css } from '@griffel/react';
```

Comment:
```suggestion
import { __css } from '@griffel/react';
```

Reply:
It will be removed by tree shaking, I don't want to remove imports as something could go wrong :)